pmm-tests/pmm-2-0-bats-tests/ps-specific-tests.bats

```diff
@@ -77,7 +77,7 @@ echo "$output"
     run pmm-admin add mysql --help
     echo "$output"
     [ "$status" -eq 0 ]
-    echo "${output}" | grep "disable-tablestats-limit=DISABLE-TABLESTATS-LIMIT"
+    echo "${output}" | grep "disable-tablestats-limit=UINT-16"

```

Comment:
this is actually a bug, UINT -16 makes no sense to me :)

Reply:
@michal-kralik would you be kind enough to clarify this? 
I suppose it similar to "STRING" and means that user need to pass unsigned int value to disable-tablestats-limit flag